auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/model/Entitlement.java

```diff
@@ -117,5 +117,6 @@ public enum Entitlement {
     CDP_GCP_RAZ,
     CDP_FEDRAMP_EXTERNAL_DATABASE_FORCE_DISABLED,
     CDP_AZURE_DATABASE_FLEXIBLE_SERVER,
-    CDP_CB_SECRET_ROTATION
+    CDP_CB_SECRET_ROTATION,
+    DATALAKE_HORIZONTAL_SCALE
```

Comment:
should this be also added to mock-thunderhead?

Reply:
That will be needed if we are adding mock tests for it.